docs/installation/cluster.md

```diff
@@ -28,3 +28,45 @@ The host is registered to the etcd cluster by the `banyand-server` automatically
 - `node-host-provider=hostname` : Default. The OS's hostname is registered as the host part in the address.
 - `node-host-provider=ip` : The OS's the first non-loopback active IP address(IPv4) is registered as the host part in the address.
 - `node-host-provider=flag` : `node-host` is registered as the host part in the address.
+
+## Etcd Client Authentication
+
+etcd supports through tls certificates and RBAC based authentication for both clients to server communication. This section is intended to help users set up authentication in etcd client.
+
+### Client-to-server authentication with username/password
+
+The etcd user can be setup by the [etcd authentication guide](https://etcd.io/docs/v3.5/op-guide/authentication/)
+
+The username/password is configured in the following command:
```

Comment:
Setting a password through command-line flags when starting a Golang server can be considered a security risk. We should recommend users set through Environment Variables.